src/Material.h

```diff
@@ -16,6 +16,9 @@
 #include "log.h"
 #endif
 
+/** The minimum allowable total cross-section value for any group */
```

Comment:
Since negative sigma_t are allowed, I would recommend changing this comment to reflect that.